sdcm/utils/adaptive_timeouts/__init__.py

```diff
@@ -58,6 +58,9 @@ class Operations(Enum):
     CREATE_MV = ("create_mv", _get_soft_timeout, ("timeout",))
     MAJOR_COMPACT = ("major_compact", _get_soft_timeout, ("timeout",))
     REPAIR = ("repair", _get_soft_timeout, ("timeout",))
+    MGMT_REPAIR = ("mgmt_repair", _get_soft_timeout, ("timeout",))
+    BACKUP = ("backup", _get_soft_timeout, ("timeout",))
+    RESTORE = ("restore", _get_soft_timeout, ("timeout",))
```

Comment:
At first, the 3 new attributes are not consistent in naming. One has `MGMT_` prefix other don't have.
And at second, 2 out of 3 attributes are not used. So, are you sure we should add it now?

Reply:
as you can see in the line above `MGMT_REPAIR`, there is already a repair, and that is the reason to have `MGMT_*` before the "new" repair... other operations today can't be done without the manager, hence it is not necessary to have the prefix to them.

the other 2 are not in use, because they will be added in following up PRs, but since i'm already changing it here, and to avoid future conflicts, i added all the necessary attributes in here at once.

Reply:
anyhow, why this PR is adding options it's not using ?

when you introduce it to backup and restore you can define them...

Reply:
i foresee conflicts (because of backports) if each one of these will add the relevant parameter at different times... so, because we may already have it in there, once we will start adding such adaptive timeouts to other `mgmt_cli_test.py`